src/management-system-v2/app/api/private/[environmentId]/processes/[processId]/images/[imageFileName]/route.ts

```diff
@@ -107,9 +108,11 @@ export async function PUT(
   const readImageResult = await readImage(request);
   if (readImageResult.error) return readImageResult.error;
 
-  await saveProcessImage(processId, imageFileName, readImageResult.buffer);
+  const newImageFileName = `_image${v4()}.${readImageResult.fileType.ext}`;
```

Comment:
Why is the uuid needed?

Reply:
I add the image as a new Image instead of overwriting the current one to prevent problems with images in versioned user tasks being overwritten. I just copied the way the filename is generated from the post endpoint.

Reply:
Makes sense, but what happens to images that aren't used in any version?

Reply:
Currently they are just stored indefinitely. I think for the database implementation there is a cleanup feature for artifacts that are not referenced anymore. I did not want to reimplement this when we will be moving to the database implementation as the default in the future.